### PR TITLE
Add missing aliases to pygmt.surface

### PR DESCRIPTION
**Pull request by michaelgrund**
**Description of proposed changes**

This PR adds the common options binary, nodata, find, header, incols and skiprows to the pygmt.surface method.

Addresses #1445

Fixes #

**Reminders**

- [ ] Run `make format` and `make check` to make sure the code follows the style guide.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Add new public functions/methods/classes to `doc/api/index.rst`.
- [ ] Write detailed docstrings for all functions/methods.
- [ ] If adding new functionality, add an example to docstrings or tutorials.

**Slash Commands**

You can write slash commands (`/command`) in the first line of a comment to perform
specific operations. Supported slash commands are:

- `/format`: automatically format and lint the code
- `/test-gmt-dev`: run full tests on the latest GMT development version

### CODE REVIEW

**Review by willschlitzer — APPROVED** [review]
Looks good! All added aliases match up with the corresponding name in the docs.

**Review by seisman — APPROVED** [review]

File: pygmt/src/surface.py [hunk 1]
```diff
@@ -25,8 +25,14 @@
...
     V="verbose",
     a="aspatial",
+    b="binary",
+    d="nodata",
+    e="find",
     f="coltypes",
+    h="header",
+    i="incols",
     r="registration",
+    s="skiprows",
```
Comment (meghanrjones) [inline pygmt/src/surface.py:35]:
```suggestion
    s="skiprows",
    w="wrap",
```

File: pygmt/src/surface.py [hunk 2]
```diff
@@ -68,8 +74,14 @@ def surface(x=None, y=None, z=None, data=None, **kwargs):
...
     {V}
     {a}
+    {b}
+    {d}
+    {e}
     {f}
+    {h}
+    {i}
     {r}
+    {s}
```
Comment (meghanrjones) [inline pygmt/src/surface.py:84]:
```suggestion
    {s}
    {w}
```

**Review by meghanrjones — COMMENTED** [review]
Another common option that was missing from the issue based on gmt docs and `THIS_MODULE_OPTIONS` macro.

**Review by meghanrjones — APPROVED** [review]

**Pull request merged**